build.gradle

```diff
@@ -153,6 +153,9 @@ dependencies {
         // macOS integration
         implementation 'org.madlonkay:desktopsupport:0.6.0'
 
+        implementation 'javax.xml.bind:jaxb-api:2.3.1'
+        runtimeOnly 'com.sun.xml.bind:jaxb-impl:2.3.4'
```

Comment:
Is it really necessary because jackson introduce woodstox for implementation?

Reply:
In my understanding,   JAXB implementation is pluggable and jackson uses woodstox. so we don't need to add 'com.sun.xml:bind:jaxb-impl'.